sql/atxs/atxs.go

```
@@ -20,14 +20,14 @@ func Get(db sql.Executor, id types.ATXID) (atx *types.ActivationTx, err error) {
 			atx, err = nil, fmt.Errorf("decode %w", decodeErr)
 			return true
 		}
-		baseTickHeight := uint64(stmt.ColumnInt64(1))
-		tickCount := uint64(stmt.ColumnInt64(2))
 		v.SetID(&id)
-		v.Verify(baseTickHeight, tickCount)
 		nodeID := types.NodeID{}
 		stmt.ColumnBytes(3, nodeID[:])
 		v.SetNodeID(&nodeID)
-		atx, err = &v, nil
+
+		baseTickHeight := uint64(stmt.ColumnInt64(1))
+		tickCount := uint64(stmt.ColumnInt64(2))
+		atx, err = v.Verify(baseTickHeight, tickCount), nil
```

why ignore the error here?

---

Which error? It's a double assignment:

```golang
atx = v.Verify(baseTickHeight, tickCount)
err = nil
```

before it was

```golang
v := functionCall()
atx, err = &v, nil
```

I'll reformat the code to make it clearer